cocos/animation/marionette/animation-controller.ts

```diff
@@ -138,8 +148,8 @@ export class AnimationController extends Component {
      * @zh 获取动画图实例中下一个状态的运行状况。
      * @en Gets the running status of the next state in the animation graph instance.
      * @param layer @en Index of the layer. @zh 层级索引。
-     * @returns @en The running status of the next state. `null` is returned in case of no transition.
-     *          @zh 下一状态运作状态对象，若未在进行过渡，则返回 `null`。
+     * @returns @en The running status of the next state. `null` is returned in case of no transition or if next state is not a motion state.
+     *          @zh 下一状态运作状态对象，若未在进行过渡或下一状态不是动画状态，则返回 `null`。
      */
     public getNextStateStatus (layer: number) {
```

Comment:
Here also fix that `getNextStateStatus ` will throw if `getCurrentTransition(layer) === null`